Corona-Warn-App/src/main/java/de/rki/coronawarnapp/task/example/ExampleArguments.kt

```diff
@@ -0,0 +1,7 @@
+package de.rki.coronawarnapp.task.example
```

Comment:
Wouldn't the examples better fit into `src/test` rather than `src/main`?

Reply:
Correct, will do later.

Reply:
Will be moved once we have another task, otherwise we will have compilation errors because dagger can't construct the map for injection.